templates/README.md

```diff
@@ -1,18 +1,24 @@
 # Templates
 
-## Updating
+## Packaging
 
-When any templates are modified, `../pkged.go` should be regenerated.
+When updates are made to these templates, they must be packaged (serialized as
+a Go struture) by running `make`, and checking in the resultant `pkged.go` file.
 
-```
-go get github.com/markbates/pkger/cmd/pkger
-pkger
-```
-Generates a pkged.go containing serialized versions of the contents of 
-the templates directory, made accessible at runtime. 
+## How it works
+
+running `make` in turn installs the `pkger` binary, which can be installed via:
+`go get github.com/markbates/pkger/cmd/pkger`
+Make then invokes `pkger` before `go build`.
+
+The resulting `pkged.go` file includes the contents of the templates directory,
+encoded as a Go strucutres which is then makde available in code using an API
+similar to the standard library's `os` package.
+
+## Rationale
 
 Until such time as embedding static assets in binaries is included in the
```

Comment:
It's now that time since go 1.16 is released :)

Reply:
Sure am happy to see this has landed.  Thre will be a follow-up PR replacing pkger with //go:embed post haste